.ci/install_cri_containerd.sh

```diff
@@ -34,9 +35,9 @@ cri_containerd_version_url="https://raw.githubusercontent.com/containerd/contain
 cri_containerd_version=$(curl -sL $cri_containerd_version_url | grep "github.com/containerd/cri" | awk '{print $2}')
 
 echo "Set up environment"
-if [ "$ID" == centos ]; then
-	# Centos: remove seccomp  from runc build
-	export BUILDTAGS=${BUILDTAGS:-apparmor}
+if [ "$ID" == centos ] || [[ "$ID" =~ ^opensuse.*$ ]]; then
+	# Centos and Opensuse: remove seccomp from runc build
+	export BUILDTAGS="apparmor"
```

Comment:
I can see this is a simplification of what was already in the file, but this does look a bit confusing to me since centos uses SELinux, not apparmor. So should this line be removing `seccomp` explicitly rather than setting it to `apparmor`?

Reply:
@jodh-intel the default value of this var is `BUILDTAGS=${BUILDTAGS:-seccomp apparmor}`[1], but if I try to do export `BUILDTAGS=""`, then the value of my exported variable is not taken and it takes the default value. 


For some reason, this change that works on my local VM, doesn't work on the CI machines, runc still being built with `seccomp`.

```
22:08:59 + [[ opensuse-leap =~ ^opensuse.*$ ]]
22:08:59 + export BUILDTAGS=apparmor
22:08:59 + BUILDTAGS=apparmor
22:08:59 + go get github.com/containerd/cri
...
22:09:26 + install_from_source
22:09:26 + echo 'Trying to install containerd from source'
22:09:26 Trying to install containerd from source
22:09:26 + cd /tmp/jenkins/workspace/kata-containers-tests-opensuse-15-PR/go/src/github.com/containerd/cri
22:09:26 + git fetch
22:09:26 + git checkout f864905c93b97db15503c217dc9a43eb65670b53
22:09:27 Note: checking out 'f864905c93b97db15503c217dc9a43eb65670b53'.
22:09:27 
22:09:27 You are in 'detached HEAD' state. You can look around, make experimental
22:09:27 changes and commit them, and you can discard any commits you make in this
22:09:27 state without impacting any branches by performing another checkout.
22:09:27 
22:09:27 If you want to create a new branch to retain commits you create, you may
22:09:27 do so (now or later) by using -b with the checkout command again. Example:
22:09:27 
22:09:27   git checkout -b <new-branch-name>
22:09:27 
22:09:27 HEAD is now at f864905c Merge pull request #1420 from chavafg/topic/fix-tests-go1.13
22:09:27 + make release
22:09:27 make[1]: Entering directory '/tmp/jenkins/workspace/kata-containers-tests-opensuse-15-PR/go/src/github.com/containerd/cri'
22:09:27 🇩 _output/containerd
22:09:27 go build -o _output/containerd \
22:09:27 	-tags 'seccomp apparmor' \
22:09:27 	-ldflags '-X github.com/containerd/cri/vendor/github.com/containerd/containerd/version.Version=f864905c-TEST -extldflags "-fno-PIC -static"' \
22:09:27 	-gcflags '' \
22:09:27 	github.com/containerd/cri/cmd/containerd
22:09:58 # github.com/containerd/cri/cmd/containerd
22:09:58 /usr/local/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
22:09:58 /usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: /tmp/go-link-177951285/000008.o: in function `pluginOpen':
22:09:58 /workdir/go/src/plugin/plugin_dlopen.go:19: warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
22:09:58 /usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cannot find -lseccomp
```


[1] https://github.com/containerd/cri/blob/master/hack/install/utils.sh#L35